### DISCUSSION TITLE
how to pack a project which use pinvoke?

### DISCUSSION THREAD

**Discussion by iyouw** [NuGet/Home#12637] [General]
how to pack a project which use pinvoke? Would somebody could give a demo?

**Comment by zivkan** [comment]
If you're P/Invoking a system dll/so, then there's nothing special from NuGet's point of view, and you pack it like any other.

If you mean you have a native dll/so that you need to ship in the package along side the managed one, then this issue has links to many similar questions. Some of them have snippets that might help you:
* https://github.com/NuGet/docs.microsoft.com-nuget/issues/2070
